monitoring-dashboard/components/org.wso2.dashboard.security.user.core/src/main/java/org/wso2/dashboard/security/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -151,9 +151,7 @@ public ReadOnlyLDAPUserStoreManager(RealmConfiguration realmConfig, ClaimManager
             this.dataSource = DatabaseUtil.getRealmDataSource(realmConfig);
         } catch (Exception ex) {
             // datasource is not configured
-            if (log.isDebugEnabled()) {
-                log.debug("Datasource is not configured for LDAP user store");
-            }
+            log.debug("Datasource is not configured for LDAP user store");
```

Comment:
We omit printing exception here. Is that intentional?

Reply:
Added now.